### DISCUSSION TITLE
Add a weather radar

### DISCUSSION THREAD

**Discussion by PinguDEV-original** [breezy-weather/breezy-weather#1187] [Ideas]
### Describe your idea

It would be cool, to have a weather radar like basically every other app ;)

### Other details

In the Bright Sky api, this feature is new: https://brightsky.dev/docs/#/operations/getRadar

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.5 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Duplicate

https://github.com/breezy-weather/breezy-weather/blob/main/docs/RADAR.md

Having such a big feature only for rainfall, only for the next 2 hours, only for Germany is way too much specific, especially when there are great free worldwide alternatives with more features available out there.
